app/lib/main.directories.g.dart

```diff
@@ -10,16 +10,17 @@
 
 // ignore_for_file: no_leading_underscores_for_library_prefixes
 import 'package:flex_ui/widgets/app_bar/app_bar.dart' as _i2;
-import 'package:flex_ui/widgets/banner/banner.dart' as _i3;
-import 'package:flex_ui/widgets/buttons/button.dart' as _i4;
-import 'package:flex_ui/widgets/cards/productCard/product_card.dart' as _i8;
-import 'package:flex_ui/widgets/cards/productCard/shared/price.dart' as _i6;
+import 'package:flex_ui/widgets/banner/banner.dart' as _i4;
```

Comment:
please tell me this is auto generated :|

not sure i have the brain capacity to hold the _i values in memory 😵

Reply:
yea anything .g. is generally code gen, so you're safe here!